scripts/bib2md.py

```diff
@@ -0,0 +1,163 @@
+# Install the library before, with the following command:
+# pip install bibtexparser --pre
+
+# to run on the CLI:
+# python bib2md.py -f reference_file.bib -ord desc|asc
+
+import bibtexparser
+import sys
+import argparse
+
+def format_simple(entry_str, order):  
+    library = bibtexparser.parse_string(entry_str)
+    formatted_entries = []
+    unprocessed_entries = []
+    
+    for entry in library.entries:
+        try:
+            authors = entry['author'].split(' and ')
+            if len(authors) > 1:
+                authors[-1] = 'and ' + authors[-1]
+              
+            authors_formatted = ', '.join([a.replace('\n', ' ').strip() for a in authors])
+            title = entry['title']
+            year = int(entry['year'])
+            venue = entry.get('journal') or entry.get('booktitle') or entry.get('archivePrefix')
+            
+            if not venue:
+                id_unprocessed = "[" + entry.key + " - " + entry.entry_type + "]"
+                unprocessed_entries.append(id_unprocessed)
+                continue
+            
+            formatted_entries.append((year, f"{authors_formatted}. {title}. {venue.value}. ({year})."))
+
+        except KeyError as e:
+            print(f"One or more necessary fields {str(e)} not present in this BibTeX entry.")
+            continue

```

Comment:
All this loop body should be a separate function that takes a single entry as input and returns a string as output.